projects/1.18/assets/create-stuff-additions/create_sa/lang/zh_cn.json

```
@@ -0,0 +1,83 @@
+  "item.create_sa.copper_leggings": "铜护腿",
+  "item.create_sa.brass_drill_head": "黄铜钻头",
+  "item.create_sa.medium_fueling_tank": "中型燃料罐",
+  "item.create_sa.brass_hoe": "黄铜锹",
+  "item.create_sa.incomplete_hydraulic_engine": "液压引擎（半成品）",
+  "item.create_sa.medium_filling_tank": "中型储水罐",
+  "item.create_sa.copper_boots": "铜靴子",
+  "item.create_sa.flamethrower": "烈焰投掷器",
+  "item.create_sa.zinc_chestplate": "锌胸甲",
+  "item.create_sa.blazing_axe": "烈焰斧",
+  "item.create_sa.slime_helmet": "黏液头盔",
+  "item.create_sa.copper_magnet": "§e铜磁铁",
+  "item.create_sa.heat_engine": "热力引擎",
+  "item.create_sa.zinc_shovel": "锌锹",
+  "item.create_sa.zinc_pickaxe": "锌镐",
+  "item.create_sa.fan_component": "风扇组件",
+  "key.create_sa.flying": "飞行",
+  "item.create_sa.copper_exoskeleton_chestplate": "铜制外骨骼",
+  "item.create_sa.blazing_cleaver": "烈焰长剑",
+  "key.create_sa.decrease_reach": "减少无人机的覆盖范围",
+  "item.create_sa.copper_shovel": "铜锹",
+  "item.create_sa.blazing_pickaxe": "烈焰镐",
+  "item.create_sa.experience_shovel": "§e经验锹",
+  "item.create_sa.brass_boots": "黄铜靴子",
+  "item.create_sa.rose_quartz_shovel": "镶金石英锹",
+  "item.create_sa.blazing_shovel": "烈焰锹",
+  "item.create_sa.large_fueling_tank": "大型燃料罐",
+  "itemGroup.tabcreate_stuff_additions_tab": "机械动力：物品附加",
+  "item.create_sa.copper_jetpack_chestplate": "铜制喷气背包",
+  "item.create_sa.experience_axe": "§e经验斧",
+  "item.create_sa.copper_hoe": "黄铜锄",
+  "item.create_sa.drone_controller": "无人机遥控器",
+  "item.create_sa.zinc_axe": "锌斧",
+  "item.create_sa.brass_exoskeleton_chestplate": "黄铜外骨骼",
+  "item.create_sa.steam_engine": "蒸汽引擎",
+  "item.create_sa.experience_pickaxe": "§e经验镐",
+  "item.create_sa.experience_sword": "§e经验剑",
+  "item.create_sa.heap_of_experience": "§e经验堆",
+  "item.create_sa.copper_chestplate": "铜胸甲",
+  "item.create_sa.andesite_jetpack_chestplate": "安山喷气背包",
+  "item.create_sa.zinc_hoe": "锌锄",
+  "item.create_sa.rose_quartz_pickaxe": "镶金石英镐",
+  "item.create_sa.small_fueling_tank": "小型燃料罐",
+  "item.create_sa.incomplete_heat_engine": "热力引擎（半成品）",
+  "item.create_sa.zinc_helmet": "锌头盔",
+  "item.create_sa.brass_axe": "黄铜斧",
+  "entity.create_sa.fan_module": "风扇组件",
+  "item.create_sa.brass_leggings": "黄铜护腿",
+  "item.create_sa.copper_axe": "铜斧",
+  "item.create_sa.zinc_handle": "锌制工具手柄",
+  "item.create_sa.rose_quartz_sword": "镶金石英剑",
+  "item.create_sa.brass_sword": "黄铜剑",
+  "item.create_sa.hydraulic_engine": "液压引擎",
+  "entity.create_sa.drill_module": "黄铜钻头",
+  "item.create_sa.zinc_boots": "锌靴子",
+  "item.create_sa.brass_chestplate": "黄铜胸甲",
+  "item.create_sa.brass_helmet": "黄铜头盔",
+  "item.create_sa.brass_drone_item": "黄铜无人机",
+  "entity.create_sa.brass_drone": "黄铜无人机",
+  "item.create_sa.rose_quartz_axe": "镶金石英斧",
+  "entity.create_sa.vault_module": "储物组件",
+  "item.create_sa.creative_filling_tank": "创造填充罐",
+  "item.create_sa.portable_drill": "便携黄铜钻头",
+  "item.create_sa.grapplin_whisk": "搅拌器抓钩",
+  "item.create_sa.block_picker": "重力拾取枪",
```

```suggestion
  "item.create_sa.block_picker": "方块拾取枪",
```
英文名和`Gravity Picker`还是略有不同（）